flowey/flowey_hvlite/src/pipelines/checkin_gates.rs

```diff
@@ -693,6 +706,7 @@ impl IntoPipeline for CheckinGatesCli {
                         artifact_dir_openhcl_igvm: ctx.publish_artifact(pub_openhcl_igvm),
                         artifact_dir_openhcl_igvm_extras: ctx
                             .publish_artifact(pub_openhcl_igvm_extras),
+                        artifact_openhcl_verify_size_baseline: publish_baseline_artifact,
```

Comment:
you shouldn't modify this existing `build_and_publish_openhcl_igvm_from_recipe` node to also serve "double-duty" to build this verify-size artifact.

simply add a new conditional `dep_on(flowey_lib_hvlite::_jobs::build_and_publish_openvmm_hcl_baseline)` that hangs off this `pipeline.new_job()`, and avoid this messy manual plumbing at the Job level.